game_api.txt

```diff
@@ -338,6 +348,104 @@ To use it, add the `on_screwdriver` function to the node definition.
  * use `on_rotate = screwdriver.disallow` to always disallow rotation
  * use `on_rotate = screwdriver.rotate_simple` to allow only face rotation
 
+
+Sfinv API
+---------
+
+### sfinv Methods
+
+* sfinv.set(player, context) - builds page formspec and calls set_inventory_formspec()
+                               if context is nil, it is either found or created.
+* sfinv.get_formspec(player, context) - builds current page's formspec
+* sfinv.get_nav_fs(player, context, vars, nav, current_idx) - see above
+* sfinv.get_layout_fs(player, context, vars) - see "customising themes" below
+* sfinv.register_page(name, def) - register a page, see below
+
+### sfinv Members
+
+* pages - table of pages[pagename] = def
+* pages_unordered - array table of pages in order of addition.
+* homepage_name - name of default page
+* contexts - contexts[playername] = player_context
+
+### Context
+
+A table with these keys:
+
+* page - current page name
+
+### sfinv.register_page
+
+sfinv.register_page(name, def)
+
+def is a table containing:
+
+* `title` - human readable page name (required)
+* `get(self, player, context, vars)` - returns a formspec string. See formspec variables. (required)
+* `is_in_nav(self, player, context)` - return true if it appears in tab header
```

Comment:
This looks a bit confusing, how about `Return true to show it in the tab header`?